pyfakefs/fake_filesystem.py

```diff
@@ -3446,19 +3445,7 @@ def islink(self, path: AnyPath) -> bool:
         return self._is_of_type(path, S_IFLNK, follow_symlinks=False)
 
     def isjunction(self, path: AnyPath) -> bool:
-        """Determine if path identifies a junction.
-
-        Args:
-            path: Path to filesystem object.
-
-        Returns:
-            `False` on posix systems.
-            `True` if path is a junction on Windows.
-
-        Raises:
-            TypeError: if path is None.
-        """
-        # TODO: implement junction on Windows
+        """Junction are never faked."""
```

Comment:
Typo: Junction -> Junctions. 
You may also add something like "Returns `False` - junctions are never faked."